pkg/hooks/webhook_test.go

```diff
@@ -1,17 +1,75 @@
 package hooks
 
 import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
 	"metacontroller/pkg/controller/common"
+	v11 "metacontroller/pkg/controller/composite/api/v1"
+	"metacontroller/pkg/logging"
+	"net/http"
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
doesn't matter much because it is a test but could use  `metav1`

Reply:
renamed